### DISCUSSION TITLE
Why are regions blocked?

### DISCUSSION THREAD

**Discussion by iwvw** [IviriusCommunity/Rebound#7] [Q&A] [answered]
_**I don’t know why the author is so arrogant, this violates the open source spirit of the community**_

![image](https://github.com/user-attachments/assets/cee66244-4c5f-49de-b0a5-d12f1524ab13)

**Comment by IviriusMain** [comment, marked as answer]
I thought this issue was fixed? Is it still restricting people from accessing Rebound Hub in those regions?
